langlib/lang.string/src/main/ballerina/string.bal

```diff
@@ -149,6 +211,10 @@ public isolated function startsWith(string str, string substr) returns boolean =
 
 # Tests whether a string ends with another string.
 #
+# ```ballerina
+# "Welcome to the Ballerina programming language".endsWith("language") ⇒ true
```

Comment:
How about adding an example resulting in `false`? (ignorable)

Reply:
I felt this case ignorable. Hence not covered.